src/cljs/rems/spa.cljs

```diff
@@ -271,28 +281,48 @@
   (rf/dispatch [:set-active-page :applications]))
 
 (secretary/defroute "/administration" []
-  (rf/dispatch [:rems.administration/enter-page])
+  (rf/dispatch [:rems.administration.administration/enter-page])
   (rf/dispatch [:set-active-page :administration]))
 
-(secretary/defroute "/create-catalogue-item" []
-  (rf/dispatch [:rems.administration.catalogue-item/enter-page])
-  (rf/dispatch [:set-active-page :create-catalogue-item]))
+(secretary/defroute "/administration/catalogue-items" []
+  (rf/dispatch [:rems.administration.catalogue-items/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/catalogue-items]))
 
-(secretary/defroute "/create-form" []
-  (rf/dispatch [:rems.administration.form/enter-page])
-  (rf/dispatch [:set-active-page :create-form]))
+(secretary/defroute "/administration/forms" []
+  (rf/dispatch [:rems.administration.forms/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/forms]))
 
-(secretary/defroute "/create-license" []
-  (rf/dispatch [:rems.administration.license/enter-page])
-  (rf/dispatch [:set-active-page :create-license]))
+(secretary/defroute "/administration/resources" []
+  (rf/dispatch [:rems.administration.resources/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/resources]))
 
-(secretary/defroute "/create-resource" []
-  (rf/dispatch [:rems.administration.resource/enter-page])
-  (rf/dispatch [:set-active-page :create-resource]))
+(secretary/defroute "/administration/workflows" []
+  (rf/dispatch [:rems.administration.workflows/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/workflows]))
 
-(secretary/defroute "/create-workflow" []
-  (rf/dispatch [:rems.administration.workflow/enter-page])
-  (rf/dispatch [:set-active-page :create-workflow]))
+(secretary/defroute "/administration/licenses" []
+  (rf/dispatch [:rems.administration.licenses/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/licenses]))
+
+(secretary/defroute "/administration/create-catalogue-item" []
+  (rf/dispatch [:rems.administration.create-catalogue-item/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/create-catalogue-item]))
+
+(secretary/defroute "/administration/create-form" []
+  (rf/dispatch [:rems.administration.create-form/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/create-form]))
+
+(secretary/defroute "/administration/create-license" []
+  (rf/dispatch [:rems.administration.create-license/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/create-license]))
+
+(secretary/defroute "/administration/create-resource" []
+  (rf/dispatch [:rems.administration.create-resource/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/create-resource]))
+
+(secretary/defroute "/administration/create-workflow" []
+  (rf/dispatch [:rems.administration.create-workflow/enter-page])
+  (rf/dispatch [:set-active-page :rems.administration/create-workflow]))
```

Comment:
`rems.navbar/navbar-items` needs to be updated for the administration link to be highlighted correctly. Rename the old page IDs and add the new ones.

Reply:
:+1: